src/useTrail.js

```diff
@@ -36,7 +36,7 @@ export const useTrail = (length, props) => {
           attach: attachController && (() => attachController),
         }
       }),
-    [length, updateProps.reverse]
+    [length, updateProps.config]
```

Comment:
This `useEffect` call inside `useTrail` wasn't using `updateProps.reverse`, but it *was* using `updateProps.config`, so I fixed the deps array.